starfish/core/spots/DecodeSpots/test/test_nearest_neighbor_trace_builder.py

```diff
@@ -0,0 +1,167 @@
+import numpy as np
+from scipy.ndimage.filters import gaussian_filter
+
+from starfish import ImageStack
+from starfish.core.spots.DecodeSpots.trace_builders import build_traces_nearest_neighbors
+from starfish.core.spots.FindSpots import BlobDetector
+from starfish.core.types import Axes
+
+
+def traversing_code() -> ImageStack:
```

Comment:
to be really exhaustive, there should probably be a test case for testing the distance filter.

Reply:
which distance filter? the search_radius param?

Reply:
Basically to make sure that trace building deals with the `search_radius` param correctly.

Reply:
ahh got it. More then what's already in the jitter_code test where it checks that it doesn't build traces when the search radius is below the 3px distance?